src/components/Atoms/Pagination/List/List.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const List = ({
+  PageComponent,
+  pages,
+  createURL,
+  onSelect,
+  className,
+  pageComponentProps,
+  ...restProps
+}) => {
+  return (
+    <div className={className}>
```

Comment:
Can we remove this div with the class and have `ul` and `li` as styled components?

Reply:
ok, i thought it's just css classes, that we want to get-rid of, will work on that now

Reply:
Fixed